### PR TITLE
Add rejumps for jump (migration, layout)

### PR DESCRIPTION
**Pull request by Konfeta3-33**

### CODE REVIEW

File: app/models/jumps/category_b.rb [hunk 1]
```diff
@@ -1,5 +1,9 @@
 module Jumps
   class CategoryB < Jump
+    def current_category
```
Comment (skyksandr) [inline app/models/jumps/category_b.rb:3]: This could be deleted

File: app/controllers/students/jumps_controller.rb [hunk 2]
```diff
@@ -47,8 +41,15 @@ def set_instructor
...
-      @student.jumps.find_or_create_by!(type: @jump.next_level_category.to_s)
+    def next_level(prev, current) # rubocop:disable Metrics/AbcSize
+      if prev == 'current' && current == 'passed'
+        @student.jumps.find_or_create_by!(type: @jump.next_level_category.to_s)
+      elsif prev == 'current' && current == 'repeat'
+        @student.jumps.create!(type: @jump.class.to_s, rejump: true)
+      elsif prev != 'current'
+        flash[:notice] = I18n.t('general.changes_updated')
+        @student.jumps.find_or_create_by!(type: @jump.next_level_category.to_s)
+      end
```
Comment (skyksandr) [inline app/controllers/students/jumps_controller.rb:52]: I suggest pass to this method `jump`, not statuses and return jump to redirect to
```
    def next_level(jump) # rubocop:disable Metrics/AbcSize
       if prev == 'current' && current == 'passed'
         @student.jumps.find_or_create_by!(type: @jump.next_level_category.to_s)
       elsif prev == 'current' && current == 'repeat'
         @student.jumps.create!(type: @jump.class.to_s, rejump: true)
       elsif prev != 'current'
         flash[:notice] = I18n.t('general.changes_updated')
         jump
       end
```

**Review by skyksandr — APPROVED** [review]
see feedback

File: app/controllers/students/jumps_controller.rb [hunk 3]
```diff
@@ -64,6 +65,7 @@ def jump_params
         :status,
         :instructor_id,
         :notes,
+        :rejump,
```
Comment (skyksandr) [inline app/controllers/students/jumps_controller.rb:68]: Doesn't look that it is needed here. 
Can not imagine case when user will need to check/uncheck this attribute

File: app/models/jumps/category_b.rb (hunk 1 above)
Comment (Konfeta3-33) [reply app/models/jumps/category_b.rb:3]: done

**Review by skyksandr — APPROVED** [review]

**Pull request merged**